ada_2_fhir-r4/zibs2020/payload/0.7-dev/nl-core-CareAgreement.xsl

```diff
@@ -0,0 +1,147 @@
+            <CarePlan>
+                <xsl:call-template name="insertLogicalId"/>
+                <meta>
+                    <profile value="http://nictiz.nl/fhir/StructureDefinition/nl-core-CareAgreement"/>
+                </meta>
+                <!-- Follows implicit zib mapping to StartDate and EndDate as described in the profile. -->
+                <status>
+                    <xsl:attribute name="value" select="'active'"/>
+                </status>
+                <!-- Normally 'plan' for zib CareAgreement, as described in the profile. -->
+                <intent value="plan"/>
+                <category>
+                    <coding>
+                        <system value="http://snomed.info/sct"/>
+                        <code value="288834001"/>
+                        <display value="Agreeing on care plan"/>
+                    </coding>
+                </category>
+                <xsl:call-template name="makeReference">
+                    <xsl:with-param name="in" select="$subject"/>
+                    <xsl:with-param name="wrapIn" select="'subject'"/>
+                </xsl:call-template>
+                <xsl:if test="begin_datum | eind_datum">
+                    <period>
+                        <xsl:call-template name="startend-to-Period">
+                            <xsl:with-param name="start" select="begin_datum"/>
+                            <xsl:with-param name="end" select="eind_datum"/>
+                        </xsl:call-template>
+                    </period>
+                </xsl:if>
+                <xsl:for-each select="$careCoordinator">
+                    <author>
+                        <xsl:if test="zorgverlener">
+                            <xsl:call-template name="makeReference">
+                                <xsl:with-param name="in" select="$careCoordinator"/>
+                                <xsl:with-param name="profile" select="'nl-core-HealthProfessional-PractitionerRole'"/>
+                            </xsl:call-template>
+                        </xsl:if>
+                        <xsl:if test="contactpersoon">
+                            <xsl:call-template name="makeReference">
+                                <xsl:with-param name="in" select="$careCoordinator"/>
+                                <xsl:with-param name="profile" select="'nl-core-ContactPerson'"/>
+                            </xsl:call-template>
+                        </xsl:if>
+                        <xsl:if test="patient">
+                            <xsl:call-template name="makeReference">
+                                <xsl:with-param name="in" select="$careCoordinator"/>
+                                <xsl:with-param name="profile" select="'nl-core-Patient'"/>
+                            </xsl:call-template>
+                        </xsl:if>
+                    </author>
+                </xsl:for-each>
+                <xsl:for-each select="$agreementParties">
```

Comment:
Here I'd expect the if statements above. In line 101 `$agreementParties` should be replaced by `.`.